src/components/calcite-tile-select-group/calcite-tile-select-group.tsx

```diff
@@ -0,0 +1,16 @@
+import { Component, Host, h } from "@stencil/core";
+
+@Component({
+  tag: "calcite-tile-select-group",
+  styleUrl: "calcite-tile-select-group.scss",
+  shadow: true,
+})
+export class CalciteTileSelectGroup {
```

Comment:
This component feels kind of empty. Do we need this level of hierarchy if there is no state?

Reply:
Yep, totally understandable.  I felt this component was necessary to get the height of all the boxes to line up regardless of their content.  The only thing it provides right now is flex-wrap display that accomplishes this feature and overrides the margins a bit to make the borders all line up like they should.  @bstifle requested we add a layout prop for 2-up 4-up, 6-up, etc. display modes, so this would be a candidate feature of `calcite-tile-select-group` if we all decide that makes sense to implement.  I tried as much as possible to just use `inline-block` style default layout because it's out-of-the-box responsive and doesn't require any other custom CSS work to make it so.  I'd like to discuss this further with the team to get other's input.

Reply:
What would 2-up etc look like in practice? Does that mean every possible integer 2-6 or just even ones? I wonder if a "fill space" and default "content width" options would make sense for container sizing, like what radio group offers with `width=full`?

<img width="618" alt="Screen Shot 2020-07-23 at 9 08 52 AM" src="https://user-images.githubusercontent.com/4733155/88310129-271ba180-ccc4-11ea-8e13-a6d66f09d360.png">

Reply:
@macandcheese I like where you're going with the idea of having a full-width and normal width option.  The 2-up, 4-up, 6-up idea sounds complicated and perhaps overkill because the people using this new component are likely going to set a container width anyway, which will constrain the horizontal space of the tiles anyway.  Because of this, I designed it to just wrap, and because there is a `max-width` set on the Tile, this allows the tiles on successive lines to still appear "lined up" as if they were in a grid.

To go in this direction would require adding layout functionality (either with flexbox or css grid) to the `calcite-tile-select-group` component.  If we like the idea of giving people the ability to specify 2-up, 4-up etc that's likely the way I'd approach it.

Reply:
Tough one for me. I was always thinking these should go into a container and just wrap.

but having the 2,3,4,5,6 up options makes sense if you are building a page or workflow that needs some nice hierarchy.

maybe we add that functionality when needed?

Reply:
I agree, I say we add that functionality when we can see a real world justification for it.  I think most users of this component will be able to accomplish their layout goals with it as-is, since they can just set a width on a container that will make it break into a certain number of visible columns based on how many tiles they're rendering.